docs/src/documentation/01-getting-started/02-for-developers.mdx

```diff
@@ -99,8 +99,7 @@ It includes contribution guidelines and information on how to run and develop th
 We want to provide only components of the highest quality.
 We can't do that without your feedback.
 If you have any suggestions about what we can do to improve components,
-please report it directly as an [issue](https://github.com/kiwicom/orbit/issues/new/choose)
-or write to us at **\#plz-orbit** on Slack.
```

Comment:
I would keep it, but mentioning that for Kiwi.com users it's possible, like: "Kiwi.com users can also use slack **plz-orbit** slack channel